.github/workflows/node-ci-main.yml

```diff
@@ -0,0 +1,57 @@
+name: Node CI (landed on protected branch)
+
+on:
+  push:
+    branches:
+    - main
+    - changeset-release/*
+    - feature/*
+
+# Our jobs run like this to minimize wasting resource cycles:
+#   1. Prime caches for primary configuration.  This way subsequent jobs
+#      can run in parallel but rely on this primed cache.
+#   2. Coverage
+#
+# For pushes directly to a branch, we assume a PR has been used with wider
+# checks, this just makes sure our coverage data is up-to-date.
+jobs:
+  prime_cache_primary:
+    name: Prime node_modules cache for primary configuration
+    env:
+      CI: true
```

Comment:
I don't think we need to set this env var. It's always on in GitHub Actions: https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables